src/events/consumer.ts

```diff
@@ -1,37 +1,57 @@
 import { Kafka, Consumer, EachMessagePayload } from 'kafkajs';
 import { ScraperRequest } from '../interfaces';
+import logger from '../lib/logger';
 
 export class EventsConsumer {
   private kafka: Kafka;
 
   private consumer: Consumer;
 
   constructor(groupId: string) {
+    logger.debug('Initializing EventsConsumer');
     const KAFKA_BROKERS = process.env.KAFKA ? process.env.KAFKA.split(',') : ['localhost:19092'];
+    logger.debug({ KAFKA_BROKERS }, 'Kafka brokers configured');
+
     this.kafka = new Kafka({
       clientId: 'peerflix-scraper',
       brokers: KAFKA_BROKERS,
```

Comment:
[Pebbles] Do you think we could pass pino logger to the Kafka constructor? In the docs they use `winston` as a logger: https://kafka.js.org/docs/custom-logger